src/app/ApplicationTemplate.Access/ApiClients/MinimumVersion/IMinimumVersionReposiory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace ApplicationTemplate.DataAccess;
+
+/// <summary>
+/// Gets the minimum version required to use the application.
+/// This was created for mocking the IUpdateRequiredService.
```

Comment:
The line 7 is probably not true since it could probably be used with a real implementation using Firebase?

Reply:
well, this is a somewhat believable interface, I'm not sure if this will be used in the end as something other than an interface for a mock service.

Reply:
I guess it makes sense. A lot of possibilities.